dynamic_programming/Unbounded_knapsack.cpp

```diff
@@ -0,0 +1,54 @@
+/* 
+The Unbounded Knapsack problem allows you to take unlimited quantities of each item. The goal is to maximize the total value 
```

Comment:
please follow the [contribution guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)  to format the header